assets/src/scss/layout/_footer.scss

```diff
@@ -1,240 +1,239 @@
 .site-footer {
   --footer-- {
     font-family: $roboto;
-    background: $dark-blue;
+    background-color: $dark-blue;
```

Comment:
These use `background` intentionally, as it supports the same values as `background-color`, but also some more like gradients and background images.